test/jdk/com/sun/jdi/lib/jdb/JdbTest.java

```diff
@@ -94,9 +94,9 @@ public void run() {
             setup();
             runCases();
         } catch (Throwable e) {
-            jdb.log("=======================================");
-            jdb.log("Exception thrown during test execution: " + e.getMessage());
-            jdb.log("=======================================");
+            Jdb.log("=======================================");
+            Jdb.log("Exception thrown during test execution: " + e.getMessage());
+            Jdb.log("=======================================");
```

Comment:
Why not just print it to `System.out` then, eliminating the method altogether?

Reply:
> Why not just print it to `System.out` then, eliminating the method altogether?

There are a 2 of other references to log() within the Jdb class. I think the only real value log() has is in case you need to (temporarily) run a test with Jdb output directed somewhere else, or maybe you want to add a prefix to the output.

I'm ok with the fix as is, but also wouldn't object to removing log() and changing all references to System.out.println.